testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/client/KeycloakTestingClient.java

```diff
@@ -81,9 +81,9 @@ public TestingResource testing(String realm) {
     }
 
     public void enableFeature(Profile.Feature feature) {
-        Set<Profile.Feature> enabledFeatures = testing().enableFeature(feature.toString());
-        Assert.assertFalse(enabledFeatures.contains(feature));
-        ProfileAssume.updateDisabledFeatures(enabledFeatures);
+        Set<Profile.Feature> disabledFeatures = testing().enableFeature(feature.toString());
+        Assert.assertFalse(disabledFeatures.contains(feature));
+        ProfileAssume.updateDisabledFeatures(disabledFeatures);
```

Comment:
Is this intentional?

Reply:
yes, that method always returns the disabled features, so it was misnamed.